### ISSUE TITLE
Estrutura do banco de dados

### ISSUE THREAD

**Issue by zitoloco** [rpuglielli/financeiro#1]
Não seria interessante ter uma pasta com a estrutura do banco de dados?

Assim, quando for preciso montar a aplicação em qualquer servidor, teremos o necessário para fazê-lo.

**Comment by rpuglielli** [comment]
Sim, @zitoloco vou atualizar o projeto e já com a estrutura inicial do DB
